workspace.jsonc

```diff
@@ -42,90 +42,89 @@
         "@stencil/webpack": "0.0.6",
         "@svgr/webpack": "5.5.0",
         "@teambit/any-fs": "0.0.5",
-        "@teambit/base-ui.constants.storage": "0.6.10",
-        "@teambit/base-ui.css-components.pill": "0.6.10",
-        "@teambit/base-ui.css-components.roundness": "0.6.11",
-        "@teambit/base-ui.elements.dots-loader": "0.6.13",
-        "@teambit/base-ui.elements.image": "0.7.0",
-        "@teambit/base-ui.elements.label": "0.7.0",
-        "@teambit/base-ui.elements.separator": "0.7.0",
-        "@teambit/base-ui.graph.tree.indent": "0.6.10",
-        "@teambit/base-ui.graph.tree.inflate-paths": "0.7.0",
-        "@teambit/base-ui.graph.tree.recursive-tree": "0.6.10",
-        "@teambit/base-ui.graph.tree.root-node": "0.6.10",
-        "@teambit/base-ui.graph.tree.tree-context": "0.6.10",
-        "@teambit/base-ui.input.error": "0.7.1",
-        "@teambit/base-ui.layout.breakpoints": "0.6.10",
-        "@teambit/base-ui.layout.grid-component": "0.6.11",
-        "@teambit/base-ui.layout.page-frame": "0.6.10",
-        "@teambit/base-ui.loaders.loader-ribbon": "0.6.11",
-        "@teambit/base-ui.routing.link": "0.1.0",
-        "@teambit/base-ui.routing.native-link": "0.1.0",
-        "@teambit/base-ui.routing.native-nav-link": "0.0.2",
-        "@teambit/base-ui.routing.nav-link": "0.1.0",
-        "@teambit/base-ui.routing.routing-provider": "0.0.2",
-        "@teambit/base-ui.styles.flex-center": "0.0.2",
-        "@teambit/base-ui.surfaces.card": "0.7.0",
-        "@teambit/base-ui.surfaces.split-pane.hover-splitter": "0.7.4",
-        "@teambit/base-ui.surfaces.split-pane.split-pane": "0.6.11",
-        "@teambit/base-ui.text.heading": "0.7.1",
-        "@teambit/base-ui.text.muted-text": "0.7.1",
-        "@teambit/base-ui.text.paragraph": "0.7.1",
-        "@teambit/base-ui.text.text": "0.0.16",
-        "@teambit/base-ui.text.text-sizes": "0.0.4",
-        "@teambit/base-ui.text.themed-text": "0.7.1",
-        "@teambit/base-ui.theme.color-palette": "0.6.12",
-        "@teambit/base-ui.theme.colors": "0.7.0",
-        "@teambit/base-ui.theme.dark-theme": "0.7.0",
-        "@teambit/base-ui.theme.fonts.roboto": "0.6.10",
-        "@teambit/base-ui.theme.sizes": "0.6.10",
-        "@teambit/base-ui.theme.theme-provider": "0.6.13",
-        "@teambit/base-ui.utils.composer": "0.0.5",
-        "@teambit/base-ui.utils.popper-js.ignore-popper-size": "0.6.11",
-        "@teambit/base-ui.utils.popper-js.resize-to-match-reference": "0.6.11",
-        "@teambit/base-ui.utils.string.affix": "0.6.10",
-        "@teambit/base-ui.utils.sub-paths": "0.6.10",
-        "@teambit/base-ui.utils.time-ago": "0.6.10",
+        "@teambit/base-ui.constants.storage": "1.0.0",
```

Comment:
why did you update all these in a major version? why not a minor version?

Reply:
I bumped to 1.0.0 because (1) it's a breaking change, and (2) I think base-ui is stable enough to have a major version